packages/edit-site/src/components/page-pages/index.js

```diff
@@ -147,17 +139,12 @@ export default function PagePages() {
 				id: 'actions',
 				cell: ( props ) => {
 					const page = props.row.original;
-					return (
-						<PageActions
-							postId={ page.id }
-							onRemove={ () => setResetQuery() }
-						/>
-					);
+					return <PageActions postId={ page.id } />;
 				},
 				enableHiding: false,
 			},
 		],
-		[ postStatuses ]
+		[ postStatuses, setView ]
```

Comment:
Is there a reason for adding `setView`?

Reply:
eslint warning, it doesn't change anything :)

Reply:
actually, no, this is a mistake :)